style/styles.css

```diff
@@ -77,4 +177,139 @@ header > a :hover {
   justify-content: space-around;
   align-items: baseline;
   list-style-type: none;
+  margin: 5px;
+  padding: 0;
+}
+
+/* section 2 : my recent works */
+.box {
+  bottom: 0;
+  width: 320px;
+  background: #ebf0ee;
+  height: 510px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  flex-direction: column;
+}
+
+.in-box {
+  position: relative;
+  background: #fff;
+  border-radius: 8px;
+  width: 320px;
+  height: 264px;
+  color: #fff;
+  top: calc(48%);
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  gap: 16px;
+}
+
+#recent-works {
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  height: auto;
+}
```

Comment:
As mentioned in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/100a933087f033def9233c8cce29c72194391fe0/portfolio/2_mobile.md?plain=1#L51), kindly use a `display: grid` property for the container of the works section.